### PR TITLE
docs(getting-started): Correcting grammar/spelling/verbiage

### PR DESCRIPTION
**Pull request by RyanChristian4427**
Minor changes to grammar, spelling, and verbiage on the "getting started" page.

### CODE REVIEW

File: docs/docs/get-started.mdx [hunk 1]
```diff
@@ -23,7 +23,7 @@ yarn add reactjs-popup -s
 
 ## Include the component
 
-To start using `reactjs-popup` you just need to import the component from the `reactjs-popup` package.
+To start using `reactjs-popup` you just need to import the component from the package.
```
Comment (RyanChristian4427) [inline docs/docs/get-started.mdx:26]: The second "`reactjs-popup`" is just redundant.

File: docs/docs/get-started.mdx [hunk 2]
```diff
@@ -52,7 +52,7 @@ import 'reactjs-popup/dist/index.css';
 
 ## Control Popup
 
-If you need more control over your component `reactjs-popup` we provide function as child pattern to get access to close popup action.
+If you need to have more control over your component, we provide function as child pattern to get access to close popup action.
```
Comment (RyanChristian4427) [inline docs/docs/get-started.mdx:55]: I'm really not sure what "function as child pattern to get access to close popup action." is supposed to mean. It doesn't make much sense even after looking through the source. I don't know what you're trying to say there but that should be reworded.

Comment (yjose) [reply docs/docs/get-started.mdx:55]: I am trying to see that you can programmatically call the `close` function.

**Comment by yjose** [comment]
Thank you @RyanChristian4427 for your time 🙌

**Pull request merged**